src/set_theory/game/basic.lean

```diff
@@ -526,7 +526,16 @@ inductive inv_ty (l r : Type u) : bool → Type u
 | right₁ : l → inv_ty ff → inv_ty tt
 | right₂ : r → inv_ty tt → inv_ty tt
 
-instance (l r : Type u) : inhabited (inv_ty l r ff) := ⟨inv_ty.zero⟩
+instance (l r : Type u) [is_empty l] [is_empty r] : is_empty (inv_ty l r tt) :=
+⟨λ h, by { cases h with _ _ _ _ a _ a _; exact is_empty_elim a }⟩
+
+instance unique_inv_ty (l r : Type u) [is_empty l] [is_empty r] : unique (inv_ty l r ff) :=
+{ default := inv_ty.zero,
+  uniq := by { rintro (a | a | a), refl, all_goals { exact is_empty_elim a } } }
+
+@[simp] theorem default_inv_ty_eq_zero (l r : Type u) [is_empty l] [is_empty r] :
+  (default : inv_ty l r ff) = inv_ty.zero :=
+rfl
```

Comment:
It would probably make sense to extract  an `inhabited` instance, which doesn't need the `is_empty` hypotheses